dynawo/sources/Models/CPP/Controls/Voltage/ModelVoltageMeasurementsUtilities/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022-2022, RTE (http://www.rte-france.com)
```

Comment:
Not 2022-2022 but 2022

Reply:
I thought it would be a good idea to prepare for the next years :) 

Changing!